<section id="services">
    <div class="container">
        <div class="row">
            <div class="col-lg-12 text-center">
                <h2 class="section-heading">Navigating the intersection of Chemistry,
                  and Computational Biology to Explore Natural Products
                </h2>
                <hr class="primary">
            </div>
        </div>
    </div>
    <div class="container">
        <div class="row">
            <div class="col-lg-6">
                <div class="service-box">
                    <i class="fa fa-4x fa-bug wow bounceIn text-primary"></i>
                    <h3>Computional Biology in the Marine Environment</h3>
                    <!--Research scope-->
                    <p class="text-muted">
                      My current research involves understanding conditional metabolism in
                      marine endosymbiotic bacteria. This has allowed me to ask
                      questions relating to how biosynthetic gene clusters (BGCs) are maintained
                      and how they may develop across different species.
                      The relationship between conditional metabolism
                      and host-associated bacteria is generally not well understood.
                      To understand possible evolutionary processes involved,
                      I incorporate biological, informatic and analytical
                      chemistry techniques to uncover selective pressures in a particular biological
                      niche. Currently, my research is leading me to explore how
                      one might identify and compare homologous BGCs to access
                      answers surrounding these questions, specifically, regarding evolution
                      and horizontal gene transfer.
                      <!-- <br>
                      Concomitantly with my studies in conditional metabolism I
                      am spear-heading investigations into identifying and
                      characterizing rare and divergent bacterial genomes. These
                      examinations have helped me to explore how one might identify
                      proteins that are highly divergent from anything in current
                      research databases. As well as direct my attentions towards
                      augmenting current approaches to develop novel methodologies
                      allowing to uncover these uncommon proteins in these
                      understudied environments. -->
                    </p>
                </div>
            </div>
            <!-- <div class="col-lg-6"> -->
                <!-- <div class="service-box"> -->
                    <!-- <i class="fa fa-4x fa-newspaper-o wow bounceIn text-primary" data-wow-delay=".2s"></i> -->
                    <!-- <h3>Publications</h3> -->
                    <!-- Conclusions from each paper -->
                    <!-- <p class="text-muted"> My research experience has priveleged -->
                      <!-- me in working with fungi and bacteria. -->
                      <!-- <a href="https://www.researchgate.net/profile/Evan_Rees2">Research Gate</a> -->
                      <!-- <br><i>or for a one-page overview please contact me for my</i> -->
                      <!-- <br><a href="#contact" class="page-scroll">Curriculum Vitae</a> -->
                    <!-- </p> -->
                <!-- </div> -->
            <!-- </div> -->
            <div class="col-lg-6">
                <div class="service-box">
                    <i class="fa fa-4x fa-terminal wow bounceIn text-primary" data-wow-delay=".3s"></i>
                    <h3>Code</h3>
                    <!--Overview of Program Functions Library-->
                    <p class="text-muted">My bioinformatics workflows have routinely
                      sharpened my abilities to operate within python, bash, and
                       R scripting environments. My tasks include employing an
                       array of DNA annotation and alignment tools to
                       <a href="img/portfolio/glebosaBGC.jpg">annotate biosynthetic gene clusters</a>
                       in metagenome-assembled genomes. With a diligence for
                       reproducibility, I combine these tasks to build custom
                       data pipelines. My data exploration utilizes
                       <a href="img/portfolio/illustratorFigure.png">Adobe Illustrator</a>,
                       and a variety of
                       <a href="img/portfolio/RstudioFigure.png">Rstudio</a>
                       and
                       <a href="img/portfolio/optimizedAutometaDashboard2.gif">Python</a>
                       libraries to measure and visualize
                       complex biological systems. I have used
                       <a href="https://www.docker.com/">docker</a>,to suit both
                       reproducibility and high throughput computing
                       environments to handle big data in a flexible, time-effective and
                       computationally efficient manner. I have worked with
                       scheduling and resource management programs such as
                       <a href="https://slurm.schedmd.com/">Slurm</a>,
                       tools from the Cooperative Computing Lab,
                       <a href="http://ccl.cse.nd.edu/software/makeflow/">Makeflow</a> and
                       <a href="http://ccl.cse.nd.edu/software/workqueue/">Work Queue</a> and finally
                       <a href="http://research.cs.wisc.edu/htcondor/">HTCondor</a>
                     </p>
                </div>
            </div>
        </div>
    </div>
</section>
